src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5008,4 +5008,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_AttributesInLocalFuncDecl" xml:space="preserve">
     <value>Attributes are not allowed on local function parameters or type parameters</value>
   </data>
+  <data name="ERR_TypeForwardedToMultipleAssemblies" xml:space="preserve">
+    <value>Type '{0}' is forwarded to multiple assemblies: '{1}' and '{2}'</value>
```

Comment:
Good point. I was considering parity with the native compiler, but adding it should improve the user experience.